network_provisioning/tool/esp_prov/prov/network_ctrl.py

```diff
@@ -13,10 +13,14 @@ def print_verbose(security_ctx, data):
         print(f'\x1b[32;20m++++ {data} ++++\x1b[0m')
 
 
-def ctrl_reset_request(security_ctx):
+def ctrl_reset_request(network_type, security_ctx):
     # Form protobuf request packet for CtrlReset command
-    cmd = proto.wifi_ctrl_pb2.WiFiCtrlPayload()
-    cmd.msg = proto.wifi_ctrl_pb2.TypeCmdCtrlReset
+    cmd = proto.network_ctrl_pb2.NetworkCtrlPayload()
+    cmd.msg = proto.network_ctrl_pb2.TypeCmdCtrlReset
+    if network_type == 'wifi':
+        cmd.cmd_ctrl_reset.net_type = 0
+    elif network_type == 'thread':
+        cmd.cmd_ctrl_reset.net_type = 1
```

Comment:
give a error and return, if not `wifi` or `thread`

Reply:
Done